src/main/java/org/spongepowered/api/command/CommandResult.java

```diff
@@ -87,7 +83,7 @@ public static CommandResult successCount(int count) {
      * @param count The amount of blocks affected
      * @return The result
      */
-    public static CommandResult affectedBlocks(int count) {
+    static CommandResult withAffectedBlocks(int count) {
```

Comment:
Vanilla removed all this - commands just return an `int` in vanilla, which is "result".